stage0/src/main.rs

```diff
@@ -145,6 +150,64 @@ unsafe fn branch_to_image(image: Image) -> ! {
     );
 }
 
+// Setup the MPU so that we can treat the USB RAM as normal RAM, and not as a
+// peripheral. Specifically we want to clear the `DEVICE` attributes, so that
+// we can allow unaligned access.
+//
+// NB: Portions opied from `sys/kern/src/arch/arm_m.rs:apply_memory_proteciton`
```

Comment:
Might be nice to have a set of functions under `lib`?

Reply:
Good idea! I think the 2 relevant functions are likely the disabling and enabling of the MPU, but it's possible I can also add one for the middle where we config the MPU. I'll take a stab at that.

Reply:
Done